uncrater/Packet_TR_Spectrum.py

```diff
@@ -0,0 +1,55 @@
+from .PacketBase import PacketBase, pystruct
+from .utils import Time2Time
+import os, sys
+import struct
+import numpy as np
+import binascii
+
+if os.environ.get("CORELOOP_DIR") is not None:
+    sys.path.append(os.environ.get("CORELOOP_DIR"))
+
+# now try to import pycoreloop
+try:
+    from pycoreloop import appId as id
+except ImportError:
+    print("Can't import pycoreloop\n")
+    print("Please install the package or setup CORELOOP_DIR to point at CORELOOP repo.")
+    sys.exit(1)
+
+
+class Packet_Metadata(PacketBase):
```

Comment:
This doesn't look right. There is a single metadata packet for both Spectrum and TR spectrum. Don't overdefine it.